util-zk/src/test/scala/com/twitter/zk/coordination/ZkAsyncSemaphoreTest.scala

```diff
@@ -143,6 +144,28 @@ class ZkAsyncSemaphoreTest extends WordSpec with MockitoSugar with AsyncAssertio
           }
         }
       }
+
+      "numPermitsOf" should {
+        "get a node's value" in {
+          val numPermitsOfMethod = PrivateMethod[Future[Int]]('numPermitsOf)
+          withClient { zk =>
+            val sem = new ZkAsyncSemaphore(zk, "/aoeu/aoeu", 2)
+            val znode = ZNode(zk, "/testing/twitter/uuu")
+            znode.create("7".getBytes)
+            val permits: Future[Int] = sem invokePrivate numPermitsOfMethod(znode)
+            assert(permits.get() == 7)
+          }
+        }
+
+        "not error on NoNode" in {
```

Comment:
this doesn't really test what we care about, right?  which is that acquire shouldn't fail just because you have a node that doesn't exist.